cmd/collector/app/handler_test.go

```diff
@@ -48,9 +49,17 @@ func (p *possiblyErroringJaegerBatchesHandler) SubmitBatches(ctx tchanThrift.Con
 	return nil, p.err
 }
 
+type possiblyErroringZipkinBatchesHandler struct {
```

Comment:
the existing one was grammatically incorrect, so let's either fix or rename to mock